### ISSUE TITLE
Check limits on Vonage rooms

### ISSUE THREAD

**Issue by bcpierce00** [clowdr-app/clowdr#73]
The limit to Vonage rooms is actually 3,000 n-to-m streams, NOT 3,000 participants. I.e. 5 people with video on = up to 600 watchers. 20 with video on = 150 watchers, 50 with video on = 60 watchers. Watchers INCLUDES the people with video on. But then the auto-switching who has cam/mic on further complicates the calculation. We need to investigate and test scaling.

Longer term, is there a way to raise these limits for conferences that need it?

**Comment by rossng** [comment]
I don't think this is an actionable issue in and of itself, so I'm converting to a discussion.

**Comment by bcpierce00** [comment]
It's actionable in the sense that it should be documented somewhere for
organizers, in the spirit of no-touch setup.

**Comment by rossng** [comment]
I've split out one issue, large rooms: #165

**Comment by rossng** [comment]
And another, finding out whether we can raise the room size limit: #166

**Comment by rossng** [comment]
We have seen rooms working fairly well at ~30 participants. At the time this was observed, the UI limited the number of streams rendered on each client to 10.

We have some more investigation to do on how many streams most users' machines can render before falling over.
